ntbs-service/Migrations/20191030113553_ChangeDotAndCaseManagementToStatus.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class ChangeDotAndCaseManagementToStatus : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<int>(
+                name: "IsEnhancedCaseManagement",
+                table: "ClinicalDetails",
+                nullable: true,
+                oldClrType: typeof(bool),
+                oldNullable: true);
+
+            migrationBuilder.AlterColumn<int>(

```

Comment:
Other statuses have enumMapping set in ntbsContext, we probably want to do it for these too?